sonarqube/docker/Dockerfile

```diff
@@ -34,6 +35,15 @@ RUN set -x \
     && mkdir -p /opt/configuration/sonarqube \
     && chown -R sonarqube:root /opt/configuration/sonarqube
 
+# fetch certificates and store them in tmp directory
+RUN if [ -z $idpDns ]; then echo 'Skipping custom certificate!'; else \
```

Comment:
why not switching this to APP_DNS?

Reply:
and also we have introduce APP_DNS_PORT ... because clusters (could) run on a different port